File: articles/machine-learning-as-algorithmic-inference/typescript/threshold_review.ts
const tp = 80, fp = 25, tn = 140, fnCount = 35;
const total = tp + fp + tn + fnCount;
console.log(`accuracy=${((tp + tn) / total).toFixed(6)}`);
console.log(`precision=${(tp / (tp + fp)).toFixed(6)}`);
console.log(`recall=${(tp / (tp + fnCount)).toFixed(6)}`);

File: articles/machine-learning-as-algorithmic-inference/rust/threshold_review.rs
fn main() {
    let tp = 80.0;
    let fp = 25.0;
    let tn = 140.0;
    let fn_count = 35.0;
    let total = tp + fp + tn + fn_count;
    let accuracy = (tp + tn) / total;
    let precision = tp / (tp + fp).max(1.0);
    let recall = tp / (tp + fn_count).max(1.0);
    println!("accuracy={:.6}", accuracy);
    println!("precision={:.6}", precision);
    println!("recall={:.6}", recall);
}
